### PR TITLE
docs: add docs for Java client admin, DDL/DML, and stream inserts methods

### PR DESCRIPTION
**Pull request by vcrfxia**
### Description 

Fixes https://github.com/confluentinc/ksql/issues/5773

### Testing done 

Docs-only change.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/developer-guide/ksqldb-clients/java-client.md [hunk 1]
```diff
@@ -466,6 +740,53 @@ CompletableFuture<Void> allInsertsFuture =
...
+             + "    FROM transactions"
+             + "    WINDOW TUMBLING (SIZE 30 SECONDS)"
+             + "    GROUP BY card_number"
+             + "    HAVING count(*) >= 3"
+             + "    EMIT CHANGES;";
+Map<String, Object> properties = Collections.singletonMap("auto.offset.reset", "earliest");
+client.executeStatement(sql, properties).get();
+```
+
+Check contents of the anomalies table with a push query:
```
Comment (vcrfxia) [inline docs/developer-guide/ksqldb-clients/java-client.md:767]: This example currently suffers from the same issue as the regular (CLI-based) version of the tutorial, and will need the corresponding update applied: https://github.com/confluentinc/ksql/issues/5827

File: docs/developer-guide/ksqldb-clients/java-client.md [hunk 2]
```diff
@@ -389,12 +398,277 @@ KsqlObject row = new KsqlObject()
 client.insertInto("ORDERS", row).get();
 ```
 
-Tutorial Examples
------------------
+Insert new rows in a streaming fashion (streamInserts())<a name="stream-inserts"></a>
+-------------------------------------------------------------------------------------
+
+Starting with ksqlDB 0.11.0, the `streamInserts()` method allows client apps to insert new rows of
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-clients/java-client.md:404]:
```suggestion
Starting with ksqlDB 0.11.0, the `streamInserts()` method enables client apps to insert new rows of
```

**Comment by vcrfxia** [comment]
Thanks for the review @JimGalasyn ! With regards to backporting, these are new features only on master. They'll be picked up in 0.11.0 but not 0.10.x or 6.0.x.

File: docs/developer-guide/ksqldb-clients/java-client.md (hunk 1 above)
Comment (vcrfxia) [reply docs/developer-guide/ksqldb-clients/java-client.md:767]: Updated the PR with the same fix as in https://github.com/confluentinc/ksql/pull/5840, so this is now resolved.

**Pull request merged**